go/src/koding/kites/kloud/provider/azure/stack.go

```diff
@@ -250,7 +254,7 @@ func (s *Stack) injectEndpointRules(vm map[string]interface{}) {
 		endpoints = append(endpoints, vmSSH)
 	}
 
-	vm["endpoint"] = endpoints
+	vm["endpoint"] = provider.ToSlice(endpoints)
```

Comment:
Should be good for now. However, this method does not work when shadowed element is inside slice.